x-pack/test/api_integration/apis/security/privileges.ts

```diff
@@ -477,372 +479,416 @@ export default function({ getService }: KibanaFunctionalTestDefaultProviders) {
+                'saved_object:visualization/bulk_get',
+                'saved_object:visualization/get',
+                'saved_object:visualization/find',
+                'ui:catalogue/dashboard',
+                'ui:navLinks/kibana:dashboard',
                 'saved_object:timelion-sheet/bulk_get',
                 'saved_object:timelion-sheet/get',
                 'saved_object:timelion-sheet/find',
-                'ui:discover/show',
+                'saved_object:canvas-workpad/bulk_get',
+                'saved_object:canvas-workpad/get',
+                'saved_object:canvas-workpad/find',
+                'saved_object:dashboard/bulk_get',
+                'saved_object:dashboard/get',
+                'saved_object:dashboard/find',
                 'ui:dashboard/show',
-                'ui:management/kibana/settings',
-                'ui:management/kibana/indices',
-                'ui:catalogue/discover',
-                'ui:catalogue/visualize',
-                'ui:catalogue/dashboard',
+                'api:console/execute',
                 'ui:catalogue/console',
                 'ui:catalogue/searchprofiler',
                 'ui:catalogue/grokdebugger',
+                'ui:navLinks/kibana:dev_tools',
                 'ui:catalogue/advanced_settings',
+                'ui:management/kibana/settings',
                 'ui:catalogue/index_patterns',
+                'ui:management/kibana/indices',
+                'app:timelion',
                 'ui:catalogue/timelion',
+                'ui:navLinks/timelion',
+                'app:graph',
                 'ui:catalogue/graph',
+                'ui:navLinks/graph',
+                'saved_object:graph-workspace/bulk_get',
+                'saved_object:graph-workspace/get',
+                'saved_object:graph-workspace/find',
+                'app:monitoring',
                 'ui:catalogue/monitoring',
+                'ui:navLinks/monitoring',
+                'app:ml',
                 'ui:catalogue/ml',
+                'ui:navLinks/ml',
+                'app:apm',
                 'ui:catalogue/apm',
+                'ui:navLinks/apm',
+                'app:maps',
                 'ui:catalogue/maps',
+                'ui:navLinks/maps',
+                'saved_object:map/bulk_get',
+                'saved_object:map/get',
+                'saved_object:map/find',
+                'app:canvas',
                 'ui:catalogue/canvas',
+                'ui:navLinks/canvas',
+                'app:infra',
                 'ui:catalogue/infraops',
+                'ui:navLinks/infra:home',
                 'ui:catalogue/infralogging',
+                'ui:navLinks/infra:logs',
+                'app:uptime',
                 'ui:catalogue/uptime',
-                'ui:navLinks/*',
+                'ui:navLinks/uptime',
               ],
             },
           });
       });
     });
-
-    describe('GET /api/security/privileges', () => {
```

Comment:
Why was this test removed?

Reply:
It shouldn't have been.... I must've messed this up somehow. Thanks for catching this.